common-ui-compose/src/main/java/app/tivi/common/compose/gradientscrim.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.common.compose
+
+import androidx.compose.Composable
+import androidx.compose.remember
+import androidx.ui.core.Draw
+import androidx.ui.geometry.Offset
+import androidx.ui.graphics.Color
+import androidx.ui.graphics.LinearGradientShader
+import androidx.ui.graphics.Paint
+import androidx.ui.unit.toRect
+import kotlin.math.pow
+
+@Composable
+fun GradientScrim(baseColor: Color, numStops: Int = 16) {
```

Comment:
Is there a special reason of why the file names of the Compose module are lowercase?

Reply:
Not really, just a convention I've picked up. I quite like it since it's then obvious what files contain lots of top-level functions/random utils, vs a proper `class`